ppr-ui/src/store/actions/actions-model.ts

```diff
@@ -44,6 +44,14 @@ export const setAccountProductSubscribtion: ActionIF = (
   commit('mutateAccountProductSubscribtion', productSubscriptions)
 }
 
+export const setUserProductSubscriptions: ActionIF = ({ commit }, products: Array<UserProductSubscriptionIF>): void => {
+  commit('mutateUserProductSubscriptions', products)
+}
+
+export const setUserProductSubscriptionsCodes: ActionIF = ({ commit }, activeProducts: Array<ProductCode>): void => {
+  commit('mutateUserProductSubscriptionsCodes', activeProducts)
+}
+
```

Comment:
Is the latter the abbreviated version of the products? For easier access?

Reply:
Yes, correct. The latter is will hold only the Active Product Codes (for easier access) vs. All Subscribed Products (Active+Inactive).

Reply:
Very cool, thanks!